policies/technical/index.md

```diff
@@ -0,0 +1,21 @@
+---
+title: OpenSSL Technical Policies
+subtitle: |
```

Comment:
There is no point in showing First issued and Moved to GitHub here - this is now just a list - so no need for an attempt at some form of edit history for the *list*.

Reply:
I have zero issues dropping that.  Done